platform/platform_win.cpp

```diff
@@ -181,3 +181,19 @@ bool Platform::GetFileSizeByFullPath(string const & filePath, uint64_t & size)
   }
   return false;
 }
+
+void Platform::SaveToSecureStorage(string const & key, string const & value)
+{
+  // Unimplemented on this platform.
```

Comment:
I suggest you to implement a single header in platform, say:
```c++
platform/secure_storage.hpp
```
and implementations for individual platforms. And Platform should have a single method, say, GetSecureStorage(), or something like that.